.github/workflows/Build.yml

```diff
@@ -274,6 +274,7 @@ jobs:
           # Verfiy and wait till latest cloud-governance version will be updated in Pypi (timeout 900 seconds)
           timeout=900
           sleep_time=30
+          pip install setuptools
```

Comment:
Pls write it in 1 line instead of 2
`pip --no-cache-dir install setuptools cloud-governance --upgrade`